core/database.py

```
@@ -161,22 +168,38 @@ async def store_message(
-            await conn.execute(
+            result = await conn.execute(
                 """
                 DELETE FROM messages WHERE message_id = $1
                 """,
                 message_id,
             )
-            logger.debug(f"Deleted message {message_id} from database")
+            # result is like "DELETE 1" or "DELETE 0"
+            deleted_count = int(result.split()[-1]) if result else 0
+            if deleted_count > 0:
+                logger.debug(f"Deleted message {message_id} from database")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)